src/Record/Record.Model/RecordBuilder.cs

```diff
@@ -307,7 +307,8 @@ public Record Build()
         recordQuads.AddRange(_storage.Scopes.Select(CreateScopeQuad));
         recordQuads.AddRange(_storage.Describes.Select(CreateDescribesQuad));
 
-        foreach (var quad in recordQuads) _graph.LoadFromString(quad.ToTripleString());
+        var tripleString = string.Join("\n", recordQuads.Select(q => q.ToTripleString()));
+        _graph.LoadFromString(tripleString);
```

Comment:
Amazing work! This was an absolutely fantastic addition to the code base. Good work!